tests/sample/sample.vcxproj

```
@@ -44,7 +44,7 @@
     <Keyword>Win32Proj</Keyword>
     <ProjectGuid>{b4ad72e3-754e-40ca-9cea-d3f2c9170e51}</ProjectGuid>
     <RootNamespace>sample</RootNamespace>
-    <ClangFlags>-g -target bpf -O2 -Werror -I../../include -I../../external/bpftool</ClangFlags>
+    <ClangFlags>-g -target bpf -O2 -Werror -I../../include -I../../external/bpftool -I../../packages/clang.headers/include </ClangFlags>
```

nit: Should the standard headers come first, so they're searched prior to others?

```suggestion
    <ClangFlags>-g -target bpf -O2 -Werror -I../../packages/clang.headers/include -I../../include -I../../external/bpftool </ClangFlags>
```